ccd-definition/ComplexTypes/LitigationFriend.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "required",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Is the claimant a child?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "fullName",
+    "FieldType": "Text",
+    "ElementLabel": "Litigation friend's full name",
+    "FieldShowCondition": "required = \"Yes\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "differentAddress",
```

Comment:
can we change this to differentAddressToLitigant?

Reply:
How about, hasDiffferentAddressToLitigant

Reply:
ok